File: bot-api/src/main/java/gg/mineral/bot/api/inv/InventoryContainer.java
package gg.mineral.bot.api.inv;

public interface InventoryContainer {

    /**
     * Retrieves the slot at the specified index.
     *
     * @param slot
     *            the index of the slot to retrieve
     * @return the slot at the specified index
     */
    Slot getSlot(int slot);
}

File: bot-api/src/main/kotlin/gg/mineral/bot/api/inv/InventoryContainer.kt
package gg.mineral.bot.api.inv

interface InventoryContainer {
    /**
     * Retrieves the slot at the specified index.
     *
     * @param inventory
     * @param slot
     * the index of the slot to retrieve
     *
     * @return the slot at the specified index
     */
    fun getSlot(inventory: Inventory, slot: Int): Slot?
}
